### DISCUSSION TITLE
November 21, 2024 (EOY Community Celebration Call) -- 12pm Pacific Time

### DISCUSSION THREAD

**Discussion by SydneyhSmith** [PowerShell/PowerShell#24464] [Community Call]
@StartAutomating will be running this call as a community end of year celebration 

Call will be 3 eastern / 12 pacific

To join use this meet https://www.meetup.com/pacific-powershell-user-group/events/304685280/

**Comment by kilasuit** [comment]
Are we going to have a similar end of year round up [like we had last year?](https://github.com/PowerShell/PowerShell/discussions/20440)

**Comment by StartAutomating** [comment]
You're all cordially invited to the [PowerShell Birthday Party](https://www.meetup.com/pacific-powershell-user-group/events/304685280/):

---

You are cordially invited to this special event.

We'll be helping Microsoft and the PowerShell community by doing an end-of-year community call and celebrating PowerShell's recent birthday.

We'll be celebrating all that PowerShell has accomplished this year, and anything we want to celebrate or highlight from the last 18 years of PowerShell!

Bring stories, demos, and yourselves to this special virtual meeting to celebrate PowerShell. Tell us how PowerShell has changed your lives.

If you have something to share, feel free to put it on the GitHub Discussions page for the community call.

**Comment by o-l-a-v** [comment]
**Automation Account: 7.4 when?**

Dotnet v7 and thus PowerShell 7.2 was EOL-ed on 8th of November 2024:

* <https://learn.microsoft.com/en-us/powershell/scripting/install/powershell-support-lifecycle#powershell-end-of-support-dates>
* <https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core>

On PowerShell Community Call 17th of October 2024 it was said that Automation Account would support 7.4 before 7.2 went EOL. This exact phrasing was not written in the notes, but it's mentioned:

* <https://github.com/PowerShell/PowerShell-RFC/blob/master/CommunityCall/notes/20241017_Notes.md> 

PowerShell 5.1 cannot be used with Az modules in Automation Account:

* <https://github.com/Azure/azure-powershell/issues/21647>

We can either use a supported version of PowerShell that can't be used with recent versions of the Azure PowerShell modules, or use an unsupported version of PowerShell.

So, what's the ETA for 7.4? 🤔

**Reply by n-bajaj** [reply]
Hi @o-l-a-v, you can try creating PS 7.4 runbooks through the Runtime environment experience. Going forward, all new runtimes would be released in Runtime environment experience only.

**Reply by o-l-a-v** [reply]
@n-bajaj: Awesome! Can confirm I'm able to select it when creating a new runtime environment.

<img width="499" alt="image" src="https://github.com/user-attachments/assets/a927b66b-e79d-4a9c-b48f-c60874af9041">

This calls for an announcement (below URLs) and doc update. 😊

* <https://azure.microsoft.com/en-us/updates/>
* <https://learn.microsoft.com/en-us/azure/automation/whats-new>
  * <https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/automation/whats-new.md>
* <https://learn.microsoft.com/en-us/azure/automation/automation-runbook-types#powershell-runbooks>
* <https://learn.microsoft.com/en-us/azure/automation/runtime-environment-overview>

**Comment by HotCakeX** [comment]
Hi, did this call happen? I can't find it on your YouTube channel. https://www.youtube.com/@powershellanddscteamchanne5739/videos

It's already December and I'm wondering when the celebration is happening which was said previously that it'd be when PowerShell 7.5 is released.

**Reply by SydneyhSmith** [reply]
This call was put on by the community outside of our usual invite that we own so we did not record

**Reply by HotCakeX** [reply]
oh i see, thanks for replying and happy holidays!
